projects/ngx-bootstrap-icons-lib/src/lib/enums/icon-names.enum.ts

```diff
@@ -340,6 +340,12 @@ export enum IconNamesEnum {
   Bookshelf = 'bookshelf',
   /** https://icons.getbootstrap.com/icons/bootstrap-fill */
   BootstrapFill = 'bootstrap-fill',
+  /** https://icons.getbootstrap.com/icons/bootstrap-icons.css */
+  BootstrapIconsCss = 'bootstrap-icons.css',
```

Comment:
what is this?

Reply:
@avmaisak Ah, I didn't notice that, I downloaded the bootstrap icons and then ran the generation script, seems they included the css file, I can delete it and rerun the script.

Reply:
There was also a json file that I removed now